### DISCUSSION TITLE
Add Weather Icon as News and Interests in the Awaiting Reimplemented Taskbar

### DISCUSSION THREAD

**Discussion by TimMarvelHero** [valinet/ExplorerPatcher#3016] [Ideas]
is it possible to add the weather icon as news and interests when you click on it? and is it possible to make make the icon look like this in the reimplemented taskbar? 
![image](https://github.com/valinet/ExplorerPatcher/assets/152800265/c8b833a1-44ef-43a7-9d0d-5e0b1843a2aa)
it looks like a newspaper and I think that icon was in Windows 10.
And please don't forget to add the darker search box (darker search box was added in Windows 10 22H2) in the reimplemented taskbar.

**Comment by pyrates999** [comment]
I think this was already declined.

**Comment by Amrsatrio** [comment]
No, sorry.

**Reply by TimMarvelHero** [reply]
Ok, please don't forget about the darker colored search box
